src/axolotl/monkeypatch/accelerate/fsdp2.py

```
@@ -191,17 +152,223 @@
     return state_dict
 
 
-def patch_accelerate_fsdp2():
-    import accelerate
-    from accelerate.utils import fsdp_utils
+def _process_lora_module_for_fsdp(module, fsdp2_kwargs):
+    """Helper function to process LoRA modules for FSDP2."""
+    from torch.distributed.fsdp import fully_shard
+    from torch.distributed.tensor import DTensor, distribute_module
```

Ruff (0.11.9)

`torch.distributed.tensor.DTensor` imported but unused

Remove unused import

(F401)